src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -43,6 +43,7 @@ GlobalAppSettings::GlobalAppSettings() :
     _wordDelimiters{ DEFAULT_WORD_DELIMITERS },
     _copyOnSelect{ false }
 {
+    _keybindings = winrt::make_self<winrt::TerminalApp::implementation::AppKeyBindings>();

```

Comment:
... why can't the winrt::make_self declaration just go into the braces above if that's how it works?

Reply:
it could i think